test/functional/testlib.bash

```diff
@@ -4242,7 +4242,6 @@ global_teardown() {
 test_setup() {
 
 	debug_msg "No test_setup was defined, using the default one..."
```

Comment:
This message has to be changed now, we are not using a "default" environment any more.